test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -144,7 +144,7 @@ func schedulePods(config *testConfig) int32 {
 			glog.Fatalf("%v", err)
 		}
 		// 30,000 pods -> wait till @ least 300 are scheduled to start measuring.
-		// TODO Find out why sometimes there may be scheduling blips in the beggining.
+		// TODO Find out why sometimes there may be scheduling blips in the begginning.
```

Comment:
s/begginning/beginning.

Reply:
http://www.thesaurus.com/browse/beginning

Reply:
It's my fault,thanks!